src/nginxconfig/i18n/en/templates/global_sections/logging.js

```diff
@@ -28,4 +28,14 @@ import common from '../../common';
 
 export default {
     enableFileNotFoundErrorLogging: `${common.enable} file not found error logging in`,
+    logformat: 'log_format',
+    enableCloudflare: 'Cloudflare',
```

Comment:
I think we should use a short sentence here to explain what is being enabled, not just the word "Cloudflare"